example.config.yml

```diff
@@ -1,64 +1,78 @@
 version: 1
 cloud:
-  provider: aws-ecs
+  provider: aws-ecs # Are you thinking there might be multiple provider types on top of AWS? Like aws-ec2 or something?
```

Comment:
Orginally yes, but decided plan "aws" will assume ECS. EC2 will never be supported.